src/app/pages/network/interfaces/interfaces-list/interfaces-list.component.ts

```diff
@@ -29,7 +29,13 @@ export class InterfacesListComponent implements OnDestroy {
   protected entityList: any;
   protected checkChangesSubscription: any;
   public hasPendingChanges = false;
-  pending_changes_text: string; 
+  public checkinWaiting = false;
+  pending_changes_text: string;
+  pending_checkin_text: string;
+  checkin_text: string = T("Settings will revert ");
+  checkin_text_2: string = T(" seconds after commit if you do not check back in.");
```

Comment:
How does this even work?  If someone saves changes ("commit"), that means keep them, do not revert.

Reply:
Basically they are applied but not permanent until the check-in, kind of like graphics settings in some OSes, that way you don't risk locking yourself out of your box